Makefile

```diff
@@ -228,6 +228,10 @@ update_tests_results:
 	@echo "🥫 Updated expected test results with actuals for easy Git diff"
 	${DOCKER_COMPOSE_TEST} run --rm -w /opt/product-opener/tests backend bash update_tests_results.sh
 
+bash:
+	@echo "🥫 Open a bash shell in the test container"
+	${DOCKER_COMPOSE_TEST} run --rm -w /opt/product-opener backend bash
```

Comment:
That's very useful :)
I keep doing "docker exec -it po-backend-1 bash"

Reply:
Thanks. I actually wasn't intending it as a permanent change, just to make it easy for me at the time, but will keep it there :-)